thunder/core/symbol.py

```diff
@@ -366,6 +366,10 @@ class BoundSymbol(BoundSymbolInterface):
     _object_ctx: dict = field(default_factory=dict)
     _executor: None | Any = None
 
+    # Only used by prims.python_return
+    # prims.copy_'s must be placed before the return statement
+    _copy_outputs_before_return: Sequence[BoundSymbol] = ()
```

Comment:
This should be `_hidden_dependencies`, right?

Reply:
Could you please explain in the comment above for future readers why this attribute is useful and where it could be applicable in the future besides prims.python_return?

Reply:
I updated the comment, but I wonder what it can be helpful for other than copy_ and return. Possibly something related to torch.distributed?